docker-compose.yml

```diff
@@ -31,14 +31,14 @@ services:
   qdrant_secondary:
     image: "qdrant/qdrant"
     volumes:
-      - dustvolume:/qdrant_secondary
+      - dustvolume:/qdrant/storage
     environment:
       QDRANT__CLUSTER__ENABLED: "true"
     ulimits:
       nofile:
         soft: 64000
         hard: 64000
-    command: ["./qdrant", "--bootstrap", "http://qdrant_primary:6335"]
+    command: ["./qdrant", "--bootstrap", "http://qdrant_secondary:6335"]
```

Comment:
@flvndvd wasn't sure if the 'primary' was intentional there, thought it would be a good occasion to ask?

Reply:
Yep this is needed, it's to bootstrap secondary from primary.